leaderboard-ui/src/app/individual-leaderboard/individual-leaderboard.component.ts

```diff
@@ -0,0 +1,25 @@
+import { Component, OnInit } from '@angular/core';
+
+@Component({
+  selector: 'app-individual-leaderboard',
+  templateUrl: './individual-leaderboard.component.html',
+  styleUrls: ['./individual-leaderboard.component.scss']
+})
+export class IndividualLeaderboardComponent implements OnInit {
+
+  public listOfIndividuals = [{
+    name: 'Kim Smith',
+    team: 'Party Parrot',
+    points: 110
+  }, {
+    name: 'Josh Butcher',
+    team: 'Coffee Cat',
+    points: 108
+  }];
+
+  constructor() { }
+
+  ngOnInit(): void {
```

Comment:
I know this is probably just scaffolding, but `ngOnInit` will have to change to `ngOnChanges` to handle async data. and since we're not using any state management - we'll need `rxjs` Observables` or an ES6 `Promise` chain. So `rxjs` would be like:
```
...
this.userData$ = of(this.apiService.doWhatever) // or new Observable<whatever>();
...
this.userData$.pipe(
      map((resp) => mapReponseToSomething),
      catchError((e)=> { doSomethingElseWith(e)})
).subscribe((someAsyncResponse: someTSType) => {
      this.userData = someAsyncResponse
})
```
where `this.userData` is defined in the currently empty constructor